src/IceRpc.Slice/SliceServiceAttribute.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) ZeroC, Inc.
+
+namespace IceRpc.Slice;
+
+/// <summary>Represents an attribute used to mark classes implementing Slice services.</summary>
+/// <remarks>The Slice source generator implements <see cref="IDispatcher"/> for classes marked with this attribute.
+/// </remarks>
+[AttributeUsage(AttributeTargets.Class, Inherited = false)]
```

Comment:
Reading between the lines here: `Inherited = false` because we want users to always specify explicitly with `SliceService` which classes should be extended via generated code. 

For example, in this case:
```csharp
[SliceService] internal partial class Chatbot : IGreeter { ... }

internal class SuperChatbot : Chatbot { ... }
```

We don't want the Slice Service code generator to extend SuperChatbot. And it wouldn't work anyway since SuperChatbot is not marked partial. If I want generated code for SuperChatbot, I need both partial and SliceService, as in:
```csharp
[SliceService] internal partial class SuperChatbot : Chatbot, ISuperHero { ... }
```

It would be good to clarify this point in the summary or the remarks.

Reply:
Right, the generator only needs to implement IDispatcher for the classes containing the attribute, we don't need to provide another implementation for derived classes.

I will add a comment.